stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -631,6 +631,7 @@ function(_compile_swift_files
 
   list(APPEND swift_flags "-enable-experimental-feature" "NonescapableTypes")
   list(APPEND swift_flags "-enable-experimental-feature" "LifetimeDependence")
+  list(APPEND swift_flags "-enable-experimental-feature" "InoutLifetimeDependence")
```

Comment:
Why do we need to keep adding experimental feature flags to the stdlib command line? Can't this be a non-experimental "baseline" feature?

Reply:
We need the experimental feature to ensure old compilers that recognize `LifetimeDependence` feature but don't have support for `@lifetime(&arg)` to parse the new stdlib interface containing this new syntax.

Reply:
Regarding the feature flag, I understand now that it is only meant to protect against an older compiler parsing the new stdlib interface. Any code outside the stdlib that uses MutableSpan simply needs to be updated--we don't expect that to be a problem in practice.